src/node-server/nlp/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "adaptive.reactivetrader.server.nlp",
+  "version": "0.0.1",
+  "description": "NLP Service for Reactive Trader",
+  "main": "lib/index",
+  "scripts": {
+    "start:dev": "nodemon --watch 'src/**/*.ts' --exec 'node -r ts-node/register --inspect' src/index.ts",
+    "start": "cross-env NODE_ENV=production node lib/index.js",
+    "build": "cross-env NODE_ENV=production tsc -p ."
+  },
+  "keywords": [
+    "NLP"
+  ],
+  "lint-staged": {
+    "src/**/*.{js,ts,tsx,json}": [
+      "prettier --write",
+      "git add"
+    ],
+    "src/**/*.{ts,tsx}": [
+      "tslint --fix",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "license": "MIT",
+  "dependencies": {
+    "dotenv": "^8.1.0",
+    "rxjs": "^6.5.3",
+    "uuid": "^3.3.2"
+  },
+  "devDependencies": {
+    "@types/dotenv": "^6.1.0",
+    "@types/node": "^11.9.3",
+    "@types/uuid": "^3.4.4",
+    "cross-env": "^5.2.0",
+    "husky": "^1.3.1",
+    "lint-staged": "^8.1.4",
+    "nodemon": "^1.18.10",
+    "prettier": "1.16.4",
+    "ts-node": "^8.0.2",
+    "tslint": "^5.13.0",
```

Comment:
Should we start using typesccript-eslint now because tslint is going to be deprecated pretty soon